src/components/FormJoinList/index.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { existCollectionByToken } from 'components/Utils/firestore';
+
+function FormJoinList() {
+  const history = useHistory();
+  const [userToken, setUserToken] = useState('');
+  const [loading, setLoading] = useState(false);
+  const [showErrorMessage, setShowErrorMessage] = useState(false);
+  const joinToListShared = async (e) => {
+    e.preventDefault();
+    setLoading(true);
+    const existsCollection = await existCollectionByToken(userToken);
+    setLoading(false);
+    if (userToken !== '' && existsCollection) {
+      localStorage.setItem('tcl18-token', userToken);
+      history.push('/list-view');
+    } else {
+      setUserToken('');
+      setShowErrorMessage(true);
+    }
+  };
+
+  return (
+    <div>
+      <form onSubmit={joinToListShared}>
+        <label>
+          <input
+            type="text"
+            placeholder="Enter a token"
+            value={userToken}
+            onChange={(e) => setUserToken(e.target.value)}
+            onKeyPress={() => setShowErrorMessage(false)}
+          />
+        </label>
+        <input type="submit" value="Join List" />
+      </form>
+      {loading ? (
+        <p id="tokenChecking" aria-live="polite">
+          Checking token shared.
+        </p>
+      ) : (
+        ''
+      )}
+      {showErrorMessage ? (
+        <p id="tokenInvalid" aria-live="assertive">
+          Invalid Token. Try again!.
+        </p>
```

Comment:
The purpose of the code is well understood. Adding the `aria-live` for accessibility is very valuable, I tried it with the "Screen Reader" and it worked perfectly, I even got excited, well done!

Reply:
Vero, thanks for your comments 💜. And yes, this part refers to acceptance criteria #3, if the user enters a token that is not valid, the error message is displayed, indicating that the token is not valid and can try again.

Reply:
Thank you very much for answering me, Cris. I get it, excellent!

Reply:
**_Thanks, Veronica!_** I appreciate your words. 

Well, Cristina gave you a clarification, great! Thanks, Cristina for answering! 😄 .